### ISSUE TITLE
Building issue on rPi OS 12 aarch64

### ISSUE THREAD

**Issue by stouracp** [morrownr/8821au-20210708#152]
Please make sure, the script works on rPi OS 12 64bit, the workaround I made is, to build the module on rPi OS 11 32bit.
The issue is, it cannot locate properly kernel-headers.

**Comment by morrownr** [comment]
Hi @stouracp 

Did you read FAQ.md? Look at the last item.

FYI: This is a problem with RasPiOS, not the script in this driver.

**Comment by stouracp** [comment]
There is answer for rPi 4/400, not for Zero 2W (would be irrelevant) but more important, nothing regarding 11/12 version. Still running this on 12 64-bit is not working, so, what would be a workaround, to compile the kernel module on 12 64-bit vrsion?

**Comment by morrownr** [comment]
> There is answer for rPi 4/400, not for Zero 2W...

I'll look at rewording the issue so that it says any 64 bit RasPi.

> Please make sure, the script works on rPi OS 12 64bit,

Can you be more specific about what problem you are seeing? I'm guessing because you provided basically nothing to go on.

**Comment by stouracp** [comment]
interesting, no idea what have changedm, but today the compiling was succesful.

**Comment by morrownr** [comment]
Is the adapter working?

FYI: There should soon by in-kernel support for your 8811au adapter. I have been helping the primary developer and the driver has been submitted to linux-wireless. It is under review currently.

**Comment by stouracp** [comment]
Yes, the adapter is working, well, I have two in Zero 2W. Both are working.
